google-cloud-asset/synth.py

```diff
@@ -133,5 +133,16 @@ def escape_braces(match):
         ])
     )
 
+s.replace(
+    'google-cloud-asset.gemspec',
+    '"README.md"',
+    '"README.md", "AUTHENTICATION.md"'
+)
+s.replace(
+    '.yardopts',
+    'README.md',
+    'README.md\nAUTHENTICATION.md'
```

Comment:
Similarly here, I'd suggest replacing `README.md\n` with `README.md\nAUTHENTICATION.md\n`

Reply:
Updated. I also found 2 gems in which `LICENSE` was being added to `.yardopts`, and in these files I merged the older operation with this one.